### ISSUE TITLE
Enable scalafix RemoveUnusedImports in the build

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#1039]
This is currently blocked by https://github.com/scalameta/scalameta/issues/924 in addition to ability to demote/promote particular messages to info/warn/error. Currently, -Xfatal-warnings promotes unused import warnings to errors which makes it impossible to use scalafix to remove the unused imports.

Another issue that needs to be solved is how to configure the demotion/promotion of particular messages. Ideally, scalahost and sbt-messagehost should be able to use the same classification/configuration.

**Comment by xeno-by** [comment]
As discussed with @olafurpg, we wouldn't like to introduce a circular dependency between Scalameta and Scalafix builds. As a result, I'll be closing this issue.
